### PR TITLE
WIP: Add Tasks to React

### PR DESCRIPTION
**Pull request by aravindballa**
I am adding tasks to the curriculum.

- [x] Philosophy

Getting Started

- [x]   Trying React
- [x]   Create React App

Components

- [x]   Custom Components
- [x]   Rendering
- [x]   Conditional Rendering
- [x]   Data flow
- [x]   Lifecycle
- [ ]   Events

Forms

- [x] Controlled Components
- [ ] Uncontrolled Components

### CODE REVIEW

**Comment by aravindballa** [comment]
@M-ZubairAhmed Check if we have to elaborate on any of the task 🔢

**Comment by M-ZubairAhmed** [comment]
they look good so far, but lets start calling them apps instead of components

File: web-dev/Frontend/Libraries & Frameworks/Learn-React.md [hunk 1]
```diff
@@ -129,7 +134,7 @@ React is an open source javaScript library for building user interfaces by Faceb
 
 ### Tasks :
 
-* Add
+* Create a time Component which updates automatically and always shows the current time. Time should be in the format `HH:MM:SS AM/PM`.
```
Comment (M-ZubairAhmed) [inline web-dev/Frontend/Libraries & Frameworks/Learn-React.md]: Can we change this to app instead ?

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: web-dev/Frontend/Libraries & Frameworks/Learn-React.md (hunk 1 above)
Comment (M-ZubairAhmed) [reply web-dev/Frontend/Libraries & Frameworks/Learn-React.md]: and others too where component is mentioned

File: web-dev/Frontend/Libraries & Frameworks/Learn-React.md [hunk 2]
```diff
@@ -133,7 +138,7 @@ React is an open source javaScript library for building user interfaces by Faceb
 
 ### Tasks :
 
-* Add
```
Comment (M-ZubairAhmed) [inline web-dev/Frontend/Libraries & Frameworks/Learn-React.md:136]: friendly ping! you missed this i guess :-)

Comment (aravindballa) [reply web-dev/Frontend/Libraries & Frameworks/Learn-React.md:136]: Ohh yeah!

**Comment by duttakapil** [comment]
Do we need to convert this from a .md file to a series of files / folders, like we did for our Laravel guide? And by when can we deploy it to our website?

**Review by M-ZubairAhmed — APPROVED** [review]

**Pull request merged**